acceptance/acceptance_test.go

```diff
@@ -126,6 +126,15 @@ func TestLabels(t *testing.T) {
 	testscript.Run(t, testScriptParamsFor(tsEnv, "label"))
 }
 
+func TestRulesets(t *testing.T) {
```

Comment:
Following the convention of the other commands, let's change this to reflect the command name 🙂 

```suggestion
func TestRuleset(t *testing.T) {
```

Reply:
@jtmcg : are you sure this is a convention?

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L30

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L39

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L48

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L75

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L84

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L102

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L111

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L120

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L129

https://github.com/cli/cli/blob/44ee17760709bedf1cad7e452e61751489b81a33/acceptance/acceptance_test.go#L138

Reply:
Gah, you're right. I crossed the streams from [this review comment](https://github.com/cli/cli/pull/9783#discussion_r1806724709) on my `repo` PR about the _directory name_, not the test name itself. I guess that means `TestRepo` is the only one not pluralized, now 😅

Reply:
Nope, it isn't. We're like 50/50 plural vs singular

Reply:
I didn't want to address this on every PR. I have a general idea to forgo all the commands so that it's one test with a table so it's just a lowercase string version of the command so we don't go round on this every time.